File: 3691-minimum-operations-to-make-columns-strictly-increasing/3691-minimum-operations-to-make-columns-strictly-increasing.go
func minimumOperations(grid [][]int) int {
    operation := 0
    var diff int
    for i := 0; i<len(grid[0]); i++ {
        for j := 1; j<len(grid); j++ {
            if grid[j][i] <= grid[j-1][i] {
                diff = grid[j-1][i] - grid[j][i]
                operation += diff + 1
                grid[j][i] = grid[j-1][i] + 1
            }
        }
    }
    return operation
}

File: 3691-minimum-operations-to-make-columns-strictly-increasing/3691-minimum-operations-to-make-columns-strictly-increasing.py
class Solution:
    def minimumOperations(self, grid: List[List[int]]) -> int:
        answer = 0
        for i in range(len(grid[0])):
            for j in range(1,len(grid)):
                if grid[j][i] <= grid[j-1][i]:
                    answer += grid[j-1][i] - grid[j][i] + 1
                    grid[j][i] = grid[j-1][i] + 1
        return answer
